ddl/backfilling.go

```diff
@@ -684,7 +684,7 @@ func (dc *ddlCtx) writePhysicalTableRecord(sessPool *sess.Pool, t table.Physical
 		}
 	})
 
-	jc := dc.jobContext(job.ID)
+	jc := dc.jobContext(job.ID, job.ReorgMeta)
```

Comment:
Why not use `reorgInfo.NewJobContext()` instead of this? In addition, how to determine whether to use 'reorgInfo.NewJobContext()' or 'dc.jobContext(job.ID, job.ReorgMeta)'

Reply:
`reorgInfo` is a concept at the higher level of DDL job, in this function, we can only get DDL job. If we can get `reorgInfo`, we prefer to call `NewJobContext` first, because this function does not need to pass additional parameters. Otherwise, call `jobContext()`.

Reply:
But this function has the argument of  `reorgInfo *reorgInfo`, so why not just use it?

Reply:
fixed in https://github.com/pingcap/tidb/pull/46756/commits/390b696357ddf7881dceca61ccfe70c6d52ffdbd